scripts/plugins/extauth-hook-AD.py

```diff
@@ -164,6 +167,10 @@ def _apply_to_cache(self):
     def _format_item(self, item):
         # CA-363207: Pam cannot handle space, put it inside []
         if " " in item:
+            if self._backend == ADBackend.bd_pbis:
+                # PBIS relace space to +, eg "ab  cd" -> "ab++cd"
```

Comment:
```suggestion
# PBIS replaces spaces with +, eg "ab  cd" -> "ab++cd"
```

Reply:
Something happened, I still see this typo

Reply:
Maybe a problem of GitHub? The latest code has
`# PBIS relace space with "+", eg "ab cd" -> "ab++cd"`